src/main/kotlin/com/coffee/api/cafe/infrastructure/persistence/repository/CafeRepositoryImpl.kt

```
@@ -1,42 +1,138 @@
+        // CoffeeBean 조회
+        val coffeeBeansQuery = jpql {
+            select(entity(CoffeeBeanEntity::class))
+                .from(entity(CoffeeBeanEntity::class))
+                .where(path(CoffeeBeanEntity::cafe).eq(cafeEntity))
+        }
+        val coffeeBean = entityManager
+            .createQuery(coffeeBeansQuery, jpqlRenderContext)
+            .resultList
+            .firstOrNull()
+            ?.let { coffeeBeanConverter.toDomain(it) }
+            ?: throw IllegalArgumentException()
```

detekt (1.23.7)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)